package.json

```diff
@@ -9,7 +9,8 @@
   "files": [
     "dist",
     "lib",
-    "src"
+    "src",
+    "THIRD-PARTY-NOTICES"
```

Comment:
Oh omg this was a nice catch. Would we need to add this to other repos? (maybe a separate item if we do?)

Reply:
Ya, we should add this to everything we distribute. AFAIK, we only have it in answers-react-components and in slapshot-tools/packages/eslint-config-slapshot

Reply:
Do we also need it for the LICENSE file?

Reply:
Ah nvm LICENSE is automatically included